org.lflang/src/org/lflang/generator/ts/TSReactionGenerator.kt

```diff
@@ -103,43 +107,66 @@ class TSReactionGenerator(
         reaction: Reaction,
         reactPrologue: String,
         reactEpilogue: String,
-        reactFunctArgs: StringJoiner,
+        reactFuncArgs: StringJoiner,
         reactSignature: StringJoiner
     ): String {
         // Assemble reaction triggers
+        var reactCode = ""
+        var reactDeadline = "}"
```

Comment:
Why is the variable initialized with a closing curly bracket?

Reply:
I defined the `reactDeadline` as `"}"`, because when the deadline is null, then the deadline should be empty.
So, at [this line](https://github.com/lf-lang/lingua-franca/pull/1318/files#diff-5fdb14bc11c18f34a23a51db49bd847b0bc8b386cc889dff06cb825d2ab20cf5R119), I replaced the value of `reactDeadline` if the reaction's deadline isn't null.

Reply:
I think the old line `139` is actually much more self-explanatory. I.e.: 
```
${" |    "..if (reaction.deadline != null) generateDeadlineHandler(reaction, reactPrologue, reactEpilogue, reactSignature) else "}"}
```

Reply:
Thank you for your nice code suggestion! I reflected your comment in [f327fee](https://github.com/lf-lang/lingua-franca/commit/f327feef4c499641d83833eb1fd3591f20c35834).